ex14/src/game.c

```diff
@@ -5,6 +5,14 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include <signal.h>
+
+int         interrupt = 0;
```

Comment:
Can you add ```interrupt``` field to ```vm_core``` struct and use this field later instead of global var in this file?

Reply:
No, because we cannot access vm_core struct unless it is global. We cannot bind additional arguments in C, so we cannot pass vm to handler